### ISSUE TITLE
libretro port? :)

### ISSUE THREAD

**Issue by DartPower** [freeorion/freeorion#3250]
Hi there, i have a idea, you want to port to libretro?
This is will be cool, really :)

Check this also please :)
https://www.libretro.com/index.php/mission/
https://docs.libretro.com/development/cores/developing-cores/
https://www.libretro.com/index.php/powered-by-libretro/
https://docs.libretro.com/development/cores/opengl-cores/

**Comment by Oberlus** [comment]
Maybe related

> FRT is a Godot "platform" targeting single board computers. In plain English, you can export a Godot game to most of them by using FRT binaries, as long as the game has been designed with the limitation of the hardware in mind.

https://github.com/efornara/frt
